src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,15 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+    private final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (userService.findByEmail(email).getPassword().equals(password)) {
+            return true;
+        }
```

Comment:
```suggestion
        if (user.getPassword().equals(password)) {
            return true;
        }
```

Reply:
You already have a user, you don`t need use "userService.findByEmail(email)" anymore

Reply:
After this write the return condition in one line, you don`t need write return 2 times

Reply:
nox fixed